pypeit/par/pypeitpar.py

```diff
@@ -431,6 +431,11 @@ def __init__(self, frame=None, illumflatten=None, method=None): #, params=None,
         dtypes['illumflatten'] = bool
         descr['illumflatten'] = 'Use the flat field to determine the illumination profile of each slit.'
 
+        defaults['tweak_slits'] = True
+        dtypes['tweak_slits'] = bool
+        descr['tweak_slits'] = 'Use the illumination flat field to tweak the slit edges. illumflatten must be set to true for this to work'
```

Comment:
Add this to the validate method.  I.e., check that illumflatten is true when tweak_slits is true.

Reply:
Added to the validate method.